### PR TITLE
Doc: class diagrams

### PR DESCRIPTION
**Pull request by valschneider**
Output can be previewed here: https://my-lisa.readthedocs.io/en/lisa-next/kernel_tests.html

### CODE REVIEW

**Comment by valschneider** [comment]
Hmph, the `TestBundle` diagram needs plantuml & graphviz to be installed. I might just generate that locally and commit the image.

**Comment by derkling** [comment]
Generating the images locally implies you have to commit in binary images, which means we kind of lost the benefits in having just textual representations to version control, isn't it?

I probably like better the idea to add the required dependencies to the `lisa-install` script, maybe guarded by an explicit flag, something similar to what we do for the SDK.

**Comment by valschneider** [comment]
If there's no objections then I'll add these dependencies.

File: install_base_ubuntu.sh [hunk 1]
```diff
@@ -59,13 +59,20 @@ install_nodejs() {
     apt-get install -y nodejs npm
 }
 
+install_doc_extras() {
```
Comment (valschneider) [inline install_base_ubuntu.sh]: I hid the extra deps behind this. Dunno if it would make sense it install it unconditionally.

Comment (derkling) [reply install_base_ubuntu.sh]: I would keep it optional... as per the SDK in the CI loop I don't think we need those bits and thus we can save a bit of download/installation  time, isn't it?

Comment (valschneider) [reply install_base_ubuntu.sh]: Okay, I'll try to tweak the doc build to make it obvious that you need those extra bits when plantuml fails

**Pull request merged**